README.md

```diff
@@ -211,6 +211,10 @@ let ban = tr(.BananasOwner(2, "John"))
 // -> "Those 2 bananas belong to John."
 ```
 
+### Automatically replace NSLocalizedString(...) calls
+
+This [script](https://gist.github.com/Lutzifer/3e7d967f73e38b57d4355f23274f303d) can be run inside the project to transform NSLocalizedString(...) calls to the tr(...) syntax.
```

Comment:
Thanks for the PR 👍 Interesting script!

Could you just maybe:
* Use backticks around `NSLocalizedString(...)` and `tr(...)` in that sentence to make them appear as code spans
* Credit yourself in the sentence (like "The script from Lutzufer can …")

Thx!